clusters/app.ci/prow/03_deployment/ghproxy-redis.yaml

```diff
@@ -135,7 +135,7 @@ objects:
               secretKeyRef:
                 key: database-password
                 name: redis
-          image: redis:6.2.6
+          image: bitnami/redis:6.2.6
```

Comment:
should be a `quay.io` pullspec if we want ot use quay?

Reply:
updated.